api/src/certification/enrolment/infrastructure/repositories/candidate-repository.js

```diff
@@ -20,7 +21,65 @@ const findBySessionId = async function ({ sessionId }) {
   return results.map(_toDomain);
 };
 
-export { findBySessionId };
+/**
+ * @function
+ * @param {Object} params
+ * @param {number} params.certificationCandidateId
+ *
+ * @return {Candidate}
+ */
+const get = async function ({ certificationCandidateId }) {
+  const certificationCandidate = await knex('certification-candidates')
+    .where({
+      id: certificationCandidateId,
+    })
+    .first();
+
+  return _toDomain(certificationCandidate);
+};
+
+/**
+ * @function
+ * @param {Object} certificationCandidate
```

Comment:
```suggestion
 * @param {Candidate} certificationCandidate
```

Si c'est bien en entrée : api/src/certification/enrolment/domain/models/Candidate.js